modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc

```diff
@@ -0,0 +1,59 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-install-configuring-host-network-interfaces-for-subnets_{context}"]
+= Configuring host network interfaces for subnets
+
+For edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. Locating remote worker nodes in subnets often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Setting up subnets for edge computing scenarios can reduce latency for the edge and allow for enhanced scalability.
```

Comment:
```suggestion
For edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. To locate remote worker nodes in subnets, you might use different network segments or subnets for the remote worker nodes than you used for the control plane subnet and local worker nodes. You can reduce latency for the edge and allow for enhanced scalability by setting up subnets for edge computing scenarios.
```

Reply:
I feel like I stumbled through these two sentences. I wonder if they can be re-written a little to keep the user actions in focus. Starting both sentences with a gerund in a row makes for some loaded reading and its hard to keep all the details straight I think. Just a suggestion, not a merge-breaker.